language_model/tensorflow/bert/README.md

```diff
@@ -59,12 +62,13 @@ python3 create_pretraining_data.py \
 ```
 
 The generated tfrecord has 500 parts, totalling to ~365GB.
+The dataset was generated using Python 3.7.6 and tensorflow-gpu 1.15.2.
 
 # Stopping criteria
-The training should occur over a minimum of 9,600,000 samples (e.g., 150k steps x 64 global batch size). A valid submission will evaluate a masked lm accuracy >= 0.678. The requirement for both minimum step count and accuracy is due to the inherent fluctuations in accuracy wrt pretraining steps, which has a standard deviation of step count measured at ~10% to meet the targer mlm_accuracy of 0.678 on a sample of 10 training runs. Based on this data, it is expected that ~95% of runs will be >= 0.678 mlm_accuracy after training 9.6M samples using the reference code and data.  
-  
-The evaluation will be on the first 500 consecutive samples of the withheld test set. Slightly more data than was needed was withheld in the test set for experimentation to determine the stopping criteria. 500 samples was chosen to minimize evaluation cost while maintaining stability of the results.  
-  
+The training should occur over a minimum of 3,000,000 samples. A valid submission will evaluate a masked lm accuracy >= 0.712. 
```

Comment:
How does the 3M examples relate to 682,666,666 steps, train_batch_size, and num_gpus? 
Does it mean total examples evaluated were 682M * 24 * 8 = 130.9Billion?

Reply:
The 682M steps is only an input the learning rate scheduler, and it's not the number steps needed to reach convergence.
The num_train_steps here is the "opt_learning_rate_training_steps" hyperparameter, which is tunable. The value used in the example command line has NOT been tuned.